### ISSUE TITLE
image is horizontally flipped

### ISSUE THREAD

**Issue by ckrc** [jacobbo/WebEye#82]
Hi @jacobbo 
I have noticed the preview image from my integrated camera got flipped horizontally. See below. 

![Capture](https://user-images.githubusercontent.com/62122682/80805248-de8ba500-8bfa-11ea-8052-ccdc3b20b1cd.JPG)

This is from the Camera app from Windows 10 which is correct
![Capture1](https://user-images.githubusercontent.com/62122682/80805253-dfbcd200-8bfa-11ea-9bd7-ba63bf0ce2f9.JPG)

I am also testing another camera which I turn my iphone into a webcam and this is showing correctly. 
![Capture2](https://user-images.githubusercontent.com/62122682/80805255-dfbcd200-8bfa-11ea-8cb4-5669d3fdbc54.JPG)

Could you please share your thoughts?

**Comment by ckrc** [comment]
Testing WebEye with two camera, the integrated camera has the flipped image while the EpoCam hasn't. This is strange... Please help

**Comment by jacobbo** [comment]
Hello, can you send me the version of the control you are referring to?

**Comment by ckrc** [comment]
![Capture5](https://user-images.githubusercontent.com/62122682/80806671-a8502480-8bfe-11ea-97b3-5aeebffb5a91.JPG)

^ @jacobbo

**Comment by ckrc** [comment]
Just to give you a bit more info, I am using a Dell XPS 15 laptop, I think its 2018 version

**Comment by jacobbo** [comment]
I'm afraid the only quick option for you would be to flip the image from the integrated camera either by using WPF tools or Bitmap.RotateFlip.

**Comment by ckrc** [comment]
@jacobbo Yes which I have done that but I don't know if this will happen to all Integrated cameras. I want to avoid flipping the image for the corrected ones

**Comment by jacobbo** [comment]
Normally, the flipping should not be happening. Add a user configurable setting to you application to enable/disable fliping.

**Comment by ckrc** [comment]
Thanks @jacobbo 
I have already added the button to enable / disable flipping. If you are able to fix this weird issue. Please do let me know. 

Thanks for your help
